css/components/listing-pages/recipes/listing-pages-recipes.css

```diff
@@ -0,0 +1,22 @@
+/**
+ * @file
+ * This file is used to style the Views listing recipes
+ */
+.view-recipes .view-content {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: stretch;
+  justify-content: space-between;
+}
+.view-recipes article {
+  flex: 0 0 calc(25% - 1rem);
+  margin: 0 0 2rem 0;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
There are also rules about property order. It works this way but if it will be a core theme itwill be important. See: https://www.drupal.org/docs/develop/standards/css/css-formatting-guidelines#declaration-order